package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "webapps-deploy",
-  "version": "2.0.0",
+  "version": "2.0.1",
```

Comment:
Update version of this library azure-actions-appservice-rest

Reply:
Will do.

Reply:
Fixed now.